appveyor.yml

```diff
@@ -17,26 +17,42 @@ environment:
     - TARGET_ARCH: "x86"
       CONDA_PY: "27"
       PY_CONDITION: "python >=2.7,<3"
+      NPY_CONDITION: "numpy ==1.10"
```

Comment:
Again, you should not change the CIs script in staged-recipes. This can be accomplished with `skip` statements in the `meta.yaml`.